### ISSUE TITLE
读取你的伴侣的想法,看一眼他的身体

### ISSUE THREAD

**Issue by mianma1** [mianma1/mianma1.github.io#3]
读完这一主题,分析你的你永远不会浪费你的时间合作伙伴说你来识别他们的感受。 如你所知,人际沟通出现在单词的百分比很小,我们完全不知道。 无论他告诉你什么,请相信,知道真相的唯一途径——整个真理是关闭你的耳朵和心脏,当他不是说看。

据说,肢体语言是非常复杂的科学,但有几个特定的信号可以学习的地方。 看看下面只是一些基本的步骤。 然后在和你的朋友、你熟悉和信任谁能够真正匹配单词和真正的感情。 一旦你有下来,使用它们在你的伴侣。

面部表情

让我们开始与面部表情的第一步。 这是一个很好的方法告诉你的伴侣是否诚实。 请看他的眼睛。 当他的目光,这意味着与你这个人是诚实的。 除此之外,的确,你的伴侣专心地看着你的眼睛当你跟他说话可能表明一个亲密的联系。 如果有人看着你的脸,也许他们正试图找出你的意思。 对他们来说,寻找一个隐秘的议程可以他们的工作。

事实上,没有什么可以击败了眉毛沟通难以置信。 尤其是在室内覆盖眼睛的太阳镜而表明他们很痛苦和不诚实。

身体

身体的其他部位会显示一个非常有趣的故事,你的伴侣。 人渴望得到的控制问题一直试图成为最后一个进门。 在讨论,如果周围的人他走沉默,这是一个不好的迹象。 他可以无聊,抵制,甚至残酷。 如果他的手在他的臀部或交叉,这就像这样。 双手交叉在胸前,意味着他是防守,试图保护自己。 他甚至可能会生气。

如果他经常触动他的头发或耳朵边和别人谈话,他是感兴趣的。 这是一个好迹象的性趣。 一旦他真的说实话与你获得你的信任通过他的眼睛和肢体语言,他是一个诚实和忠诚的人。

总之,除了根据星座来了解你的伴侣更好,星座还提到了很多方法来确定他的诚实。

After reading this topic, you will never waste your time analyzing what your partner says to you to identify their feelings. As you know, the percent of interpersonal communication appearing through words is so small, and we don't know exactly. So, no matter what he tells you, please believe that the only way to know the truth - the entire truth is turning your ears and heart off and watching for when he is not saying.

It is said that the science of body language is very complicated, but there are several certain signals you can learn to spot. Check out just some basic steps below. Then try them out with your friends, who you know well and trust to be able to match true words and real feelings. Once you have got them down, use them on your partner.

Facial Expression

Let's start the first step with facial expression. This is a good way to inform whether or not your partner is being honest with you. Please look at his eyes. When he glances at nowhere, it means this person is honest with you. Besides, it is true that your partner looking into your eyes intently while you are talking to him may suggest an intimate connection. If someone looks at your face, maybe they are trying to find out what you mean. For them, looking for a hidden agenda can be their work.

Actually, nothing can beat raised eyebrows to communicate disbelief. Especially, covering their eyes with sunglasses while indoors is shown that they are distressed and dishonest.

The Body

The rest of the body will display a very interesting story about your partner. Someone who desires to get the control of any issue always tries to be the last one through the door. While in the discussion, if people around him go silent, that is a bad sign. He can be bored, resisting, or even cruel. If his hands are on his hips or crossed, that is like the case. His hands folded across his chest means that he is feeling defensive and attempting to guard himself. He may be even angry.

If he often touches his hair or ears while talking to someone else, he is interested. That is a good sign for the sexual interest. Once he is genuinely telling the truth with you to gain your trust through his eyes and body language, he is an honest and loyal person.

To sum up, in addition to depending on Zodiac signs to get to know your partner better, Horoscope also mentions many ways to determine his honesty.
